_targets.R

```diff
@@ -6,10 +6,40 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+nwis_Cd <- '00010'
+p_width <- 12
+p_height <- 7
+p_units <- "in"
+
 p1_targets_list <- list(
+  tar_target(
+    site_data_01427207,
```

Comment:
per data science branch conventions, let's add the file extension to the end of the names for these file targets (e.g., `site_data_01427207_csv`)

Reply:
Done!